CONTRIBUTING.md

```diff
@@ -139,9 +148,89 @@ When reporting a bug please include the following key pieces of information:
 
 ## Promote your company on kserve.io
 
-If your company supports KServe or uses in on production, you can list it on our [adopters page](https://kserve.github.io/website/master/community/adopters).
+If your company supports KServe or uses in on production, you can list it on our
+[adopters page](https://kserve.github.io/website/master/community/adopters).
 We have categories for *providers* (who offer hosted or managed KServe services
-to their customers), *end user* (who use and consume KServe) and *integrations* (commercial or open source products
-that work with KServe).
+to their customers), *end user* (who use and consume KServe) and *integrations*
+(commercial or open source products that work with KServe).
 
 Please add your company logo, preferably in SVG format, to [here](https://github.com/kserve/website/tree/main/docs/images).
+
+## Becoming a committer
+
+A committer is a contributor who has gained the privilege to commit code to the
+project. But being a committer also means being committed to the project and the
+community as a whole.
+
+**Responsibilities of a committer:**
+
+- Triage and respond to issues
+- Be an expert in certain areas, but familiar with the majority of the code base
+- Review pull requests
+- Answer questions on Slack
+- Mentor new project contributors
+
+Contributors who have demonstrated their proficiency to make substantial code
+contributions and shown their commitment to the projects success over a
+prolonged period of time are candidates to become committers.
```

Comment:
If possible, I'd recommend adding some objective metrics. That would make easier for non-committers to better understand what to do and how far they are from being a committer.

I know it is not possible to fully remove subjectivity, but some objectivity is valuable.

Example of a template: https://github.com/cncf/project-template/blob/main/GOVERNANCE-maintainer.md#becoming-a-maintainer

Reply:
...and Jaeger project using that template: https://github.com/jaegertracing/jaeger/blob/main/GOVERNANCE.md#becoming-a-maintainer

Reply:
Hi Edgar,

two good points!

> ... make easier for non-committers to better understand what to do and how far they are from being a committer.
my intention of writing this ...

It was my intention to answer this question by writing this paragraph:

> **Responsibilities of a committer:**
> 
> - Triage and respond to issues
> - Be an expert in certain areas, but familiar with the majority of the code base
> - Review pull requests
> - Answer questions on Slack
> - Mentor new project contributors
> 
> Contributors who have demonstrated their proficiency to make substantial code contributions and shown their commitment to the projects success over a prolonged period of time are candidates to become committers.

I will add some more text to make it more clear.

Reply:
> If possible, I'd recommend adding some objective metrics. 

That is why I created the `list-contributors.py` script. It produces a list of candidates based on their recent activity, like creating and reviewing PRs. 

**Top PR authors:**
```shell
$ make list-pr-authors

Contributors to 'kserve/kserve' (PR authors) by number of PRs (>3) since 2023-03-02:

 39  supercoder      [5083, 5077, 5040, 5038, 5035, 5031, 5024, 5016, 5006, 5004, ... ]
 20  kservehacker    [5089, 5061, 5054, 5029, 5020, 5002, 5001, 4998, 4956, 4952, ... ]
  6  code4fun        [5069, 5048, 5027, 5026, 5008, 4867]
```

**Top PR reviewers:**
```
$ make list-pr-reviewers

Contributors to 'kserve/kserve' (PR commenters, reviewers) by number of PRs (>3) since 2023-03-02:

 13  reviewchamp     [5089, 5075, 5072, 5047, 5042, 5040, 5038, 5031, 4993, 4969, ... ]
  9  nitpickr        [5079, 5054, 5049, 5040, 4954, 4877, 4782, 4762, 4739]
  4  casualcodr      [5062, 5039, 5025, 4867]
```


We could add more metrics in the future. I just wanted to start with something useful.